### DISCUSSION TITLE
VidPrevtyn Beta kommt demnächst

### DISCUSSION THREAD

**Discussion by TH-Pega** [Digitaler-Impfnachweis/certification-apis#251] [Ideas]
Hallo,

bitte ergänzen Sie zeitnah den neuen Sanofi Pasteur-Impfstoff "VidPrevtyn Beta", der ab Mitte Dezember ausgeliefert werden soll:

ma: ORG-100000788
mp: EU/1/21/1580
vp: J07BX03

Aktuell können dafür keine Zertifikate erstellt werden.

**Comment by kamianni** [comment]
Ist es zeitlich absehbar, ab wann auf der Referenzumgebung und der Produktivumgebung Zertifikate erzeugt werden können? Wir bräuchten diese Information für die anstehenden Tests und den Kundenkontakt.
